commitizen/git.py

```diff
@@ -39,11 +39,15 @@ class GitTag(GitObject):
     def __init__(self, name, rev, date):
         self.rev = rev.strip()
         self.name = name.strip()
-        self.date = date.strip()
+        self._date = date.strip()
 
     def __repr__(self):
         return f"GitTag('{self.name}', '{self.rev}', '{self.date}')"
 
+    @property
```

Comment:
May I know why do we need this `property`?

Reply:
In order to mock it, the original value in `self.date` is now `self._date` and becomes the internal value